src/Libraries/Game.Server/Extensions/ItemExtensions.cs

```diff
@@ -65,6 +65,34 @@ public static uint GetMaxMagicWeaponDamage(this ItemData item)
         return item.GetMaxMagicWeaponBaseDamage() + item.GetAdditionalWeaponDamage();
     }
 
+    public static uint GetHairPartClientId(ItemInstance? itemInstance, EPlayerClass playerClass)
+    {
+        if (itemInstance is null)
+        {
+            return 0;
+        }
+       
+        var itemId = itemInstance.ItemId;
+        if (itemId < 72000)
```

Comment:
Can you put this magic value (and the following below) into some constants so we know what these magic numbers refer to? I think there is no fitting class for it yet. Feel free to add a `GearConstants` or similar